setup.py

```diff
@@ -1,5 +1,5 @@
 import os
-from distutils.core import setup
+from setuptools import setup
```

Comment:
When using `pip` we don't really need this, but the setup is more "friendly" if we import is from `setuptools` from the beginning.